pytorch_lightning/trainer/properties.py

```diff
@@ -483,6 +485,19 @@ def sanity_checking(self, val: bool) -> None:
     Loop properties
     """
 
+    @property
+    def fit_loop(self) -> FitLoop:
+        # TODO(@awaelchli): the current train_loop should be renamed to fit_loop
+        return self.train_loop
+
+    @property
+    def training_epoch_loop(self) -> TrainingEpochLoop:
+        return self.fit_loop.training_loop
+
+    @property
+    def training_batch_loop(self) -> TrainingBatchLoop:
+        return self.fit_loop.training_loop.batch_loop
```

Comment:
similarly, why do these need to be exposed as properties? it opens up the surface for backward compatibility by a ton

Reply:
these two don't need to be, I can chain the calls `self.fit_loop.training_loop.batch_loop` in the tests. But at the minimum the top level fit loop should be accessible.

Reply:
> But at the minimum the top level fit loop should be accessible.

why's that? just thinking about it from a backwards compatibility/deprecation surface area point of view. could we be more selective as to which features are exposed on the trainer?

Reply:
okay, removed them. the only public access to the loop now is through `trainer.train_loop` which is what we have so far on master.